src/server/protocached.proto

```diff
@@ -27,7 +27,7 @@ message SetResponse {
     // Next id: 4
 }
 
-service ProtoCachedService {
+service Protocached {
```

Comment:
Is there a reason this needed to be renamed? Just curious if this is critical to make the grpc worked.